arch/arm/soc/atmel_sam/sam4s/Kconfig.soc

```diff
@@ -112,7 +105,15 @@ config SOC_ATMEL_SAM4S_PLLB_DIVB
 	Board config file can override this settings
 	for a particular board.
 
-	With default of MULB == 6, and DIVB == 1,
-	PLL is running at 7 times of main clock.
+	With default of MULB == 3, and DIVB == 1,
+	PLL is running at 4 times of main clock.
+
+config SOC_ATMEL_SAM4S_EFC0_FMR_FWS
```

Comment:
I don't see why it's bad to make it a configure option. A person would design a board and look at that table and choose the frequency they want to run at and appropriate VDDCORE and VDDIO. Then they would set the FWS accordingly to work with their board. It's like any other option. If we are going to change processor clock frequency on the fly then why do we even have processor speed as an option? I see this config option as the same thing as setting your PLL multiplier or divider. @galak or @nashif could you help us with this?